src/__tests__/config-loader.test.ts

```diff
@@ -97,4 +97,20 @@ describe("config-loader", (): void => {
     expect(successResult.resultType).toBe("success");
     expect(successResult.configFileAbsolutePath).toBe(configFile);
   });
+
+  it("should allow an absolute baseUrl in tsconfig.json", () => {
+    const result = configLoader({
+      explicitParams: undefined,
+      cwd: "/baz",
+      // tslint:disable-next-line:no-any
+      tsConfigLoader: (_: any) => ({
+        tsConfigPath: "/baz/tsconfig.json",
+        baseUrl: "/baz",
+        paths: {},
+      }),
+    });
+
+    const successResult = result as ConfigLoaderSuccessResult;
+    assert.equal(successResult.absoluteBaseUrl, "/baz");
```

Comment:
Oops, this is using `assert` which is undefined.

@jonaskello Would be nice to run test CI on PRs!

Reply:
Good catch! Looks like my PR predated the move to Jest in https://github.com/dividab/tsconfig-paths/pull/186. I can patch up this test if you aren't already working on that.

Reply:
I'll let you handle it, thanks!

Reply:
@aleclarson Yes, definitely would be nice to run the tests in CI :-). It used to work but perhaps it was broken by the move to github actions. I'll see if I can get that running again.

Reply:
PR with passing test suite: https://github.com/dividab/tsconfig-paths/pull/211